webnn/idlharness.https.any.js

```diff
@@ -9,8 +10,8 @@
 
 idl_test(
   ['webnn'],
-  ['html', 'WebIDL', 'webgl1', 'webgpu'],
-  idl_array => {
+  ['html', 'webidl', 'webgl1', 'webgpu'],
```

Comment:
```suggestion
  ['html', 'webidl', 'webgpu'],
```
(webnn no longer depends on WebGL)

Reply:
I've updated this PR, please take another look, thanks.